uint32 imageId
uint32 signalStrength
bool detection
bool safe
uint32[10] signalHistogram
